src/chrome/crdpMultiplexing/webSocketToLikeSocketProxy.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { logger } from 'vscode-debugadapter';
+import * as WebSocket from 'ws';
+import { LikeSocket } from 'noice-json-rpc';
+
+export class WebSocketToLikeSocketProxy {
+    private _server: WebSocket.Server;
+
+    constructor(private _port: number, private _socket: LikeSocket) { }
+
+    public start(): void {
+        this._server = new WebSocket.Server({ port: this._port }, () => {
```

Comment:
Does this have to be a websocket? Does VS actually think it's connecting to Chrome, or does it just know it needs to connect on some port?